engine/api/notification/permission.go

```diff
@@ -19,9 +19,6 @@ func projectPermissionUserIDs(ctx context.Context, db gorp.SqlExecutor, store ca
 
 	var groupIDs []int64
 	for _, g := range proj.ProjectGroups {
-		if group.DefaultGroup != nil && group.DefaultGroup.ID == g.Group.ID {
```

Comment:
Be careful removing this will allow sending notification on the "old" default group.